### PR TITLE
add ClickHouse Keeper to doc pages describing ZooKeeper use

### LINKED ISSUE #67: Replace ZooKeeper to ClickHouse Keeper in docs and tutorials.
Keep ZooKeeper as obsolete option.

### PR DESCRIPTION
**Pull request by DanRoscigno**
### Changelog category (leave one):
- Documentation (changelog entry is not required)

This PR closes https://github.com/ClickHouse/clickhouse-docs/issues/67

### CODE REVIEW

**Comment by DanRoscigno** [comment]
List of pages to verify
- [ ] http://localhost:3000/docs/en/operations/clickhouse-keeper
- [ ] http://localhost:3000/docs/en/operations/configuration-files
- [ ] http://localhost:3000/docs/en/operations/settings/merge-tree-settings
- [ ] http://localhost:3000/docs/en/operations/settings/settings
- [ ] http://localhost:3000/docs/en/operations/system-tables/distributed_ddl_queue
- [ ] http://localhost:3000/docs/en/operations/system-tables/mutations
- [ ] http://localhost:3000/docs/en/operations/system-tables/replicas
- [ ] http://localhost:3000/docs/en/operations/system-tables/replication_queue
- [ ] http://localhost:3000/docs/en/operations/system-tables/zookeeper
- [ ] http://localhost:3000/docs/en/operations/tips
- [ ] http://localhost:3000/docs/en/operations/utilities/clickhouse-copier

File: docs/en/operations/clickhouse-keeper.md [hunk 1]
```diff
@@ -5,41 +5,41 @@ sidebar_label: ClickHouse Keeper
...
 
-ClickHouse Keeper supports Access Control List (ACL) the same way as [ZooKeeper](https://zookeeper.apache.org/doc/r3.1.2/zookeeperProgrammers.html#sc_ZooKeeperAccessControl) does. ClickHouse Keeper supports the same set of permissions and has the identical built-in schemes: `world`, `auth`, `digest`, `host` and `ip`. Digest authentication scheme uses pair `username:password`. Password is encoded in Base64.
+ClickHouse Keeper supports Access Control Lists (ACLs) the same way as [ZooKeeper](https://zookeeper.apache.org/doc/r3.1.2/zookeeperProgrammers.html#sc_ZooKeeperAccessControl) does. ClickHouse Keeper supports the same set of permissions and has the identical built-in schemes: `world`, `auth`, `digest`, `host` and `ip`. The digest authentication scheme uses the pair `username:password`, the password is encoded in Base64.
```
Comment (antonio2368) [inline docs/en/operations/clickhouse-keeper.md:16]: @alesapin is it true for `host` and `ip`? I don't see it in code.

File: docs/en/operations/system-tables/replicas.md [hunk 2]
```diff
@@ -62,13 +62,13 @@ Columns:
     Note that writes can be performed to any replica that is available and has a session in ZK, regardless of whether it is a leader.
```
Comment (antonio2368) [inline docs/en/operations/system-tables/replicas.md:62]: What about `ZK`? We use ClickHouse Keeper everywhere so ZK could be confusing now?

File: docs/en/operations/system-tables/mutations.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ Columns:
 
 -   `table` ([String](../../sql-reference/data-types/string.md)) — The name of the table to which the mutation was applied.
 
--   `mutation_id` ([String](../../sql-reference/data-types/string.md)) — The ID of the mutation. For replicated tables these IDs correspond to znode names in the `<table_path_in_zookeeper>/mutations/` directory in ZooKeeper. For non-replicated tables the IDs correspond to file names in the data directory of the table.
+-   `mutation_id` ([String](../../sql-reference/data-types/string.md)) — The ID of the mutation. For replicated tables these IDs correspond to znode names in the `<table_path_in_zookeeper>/mutations/` directory in ClickHouse Keeper or ZooKeeper. For non-replicated tables the IDs correspond to file names in the data directory of the table.
```
Comment (antonio2368) [inline docs/en/operations/system-tables/mutations.md:11]:
```suggestion
-   `mutation_id` ([String](../../sql-reference/data-types/string.md)) — The ID of the mutation. For replicated tables these IDs correspond to znode names in the `<table_path_in_clickhouse_keeper>/mutations/` directory in ClickHouse Keeper. For non-replicated tables the IDs correspond to file names in the data directory of the table.
```

In other places it's completely removed so this makes more sense to me. What do you think?

File: docs/en/operations/system-tables/replicas.md (hunk 2 above)
Comment (DanRoscigno) [reply docs/en/operations/system-tables/replicas.md:62]: Good catch, I missed that.  I will search through and update.

File: docs/en/operations/clickhouse-keeper.md (hunk 1 above)
Comment (alesapin) [reply docs/en/operations/clickhouse-keeper.md:16]: Agree @antonio2368, we don't support them.

Comment (DanRoscigno) [reply docs/en/operations/clickhouse-keeper.md]: @alesapin do I need to change the wording for the ACLs to say :

ClickHouse Keeper supports a subset of the ZooKeeper shemes, `world`, `auth` and `digest` schemes are supported, but `host` and `ip` schemes are not.

**Comment by DanRoscigno** [comment]
related to https://github.com/ClickHouse/ClickHouse/issues/32513

**Pull request merged**